enum_tools/utils.py

```diff
@@ -40,6 +40,11 @@ def init_logfile(logfile, fmt):
 
         now = datetime.datetime.now().strftime("%d/%m/%Y %H:%M:%S")
         with open(logfile, 'a', encoding='utf-8') as log_writer:
+            if fmt == 'json':
+                log_writer.write(
+                    json.dumps({'start_time': datetime.datetime.now().isoformat()}) + '\n')
+                return
+
```

Comment:
Perhaps this and the following line should just be replaced with `fmt_output` for consistency?

Reply:
Yes, that is an option to do - the idea is just to print out that the process started

The whole JSON log - because of the way the code is designed, will print out single JSON lines that can be processed one by one (using the "\n" as a delimiter between json outputs)